src/popover/popover-content.component.ts

```diff
@@ -7,11 +7,20 @@ import { Component, HostBinding } from "@angular/core";
 	selector: "cds-popover-content, ibm-popover-content",
 	template: `
 		<span class="cds--popover-content">
-			<ng-content></ng-content>
+			<div>
+				<ng-content></ng-content>
+			</div>
+			<span *ngIf="autoAlign" class="cds--popover-caret cds--popover--auto-align"></span>
 		</span>
-		<span class="cds--popover-caret"></span>
+		<span *ngIf="!autoAlign" class="cds--popover-caret"></span>
 	`
 })
 export class PopoverContent {
 	@HostBinding("class.cds--popover") popoverClass = true;
+
+	/**
+	 * Helper input property
```

Comment:
Add experimental notice

Reply:
Removing helper input property since we do not want to make any API changes we may need to remove in future as Z's suggestions.